### DISCUSSION TITLE
spectral-cli.exe lint swagger2model.json on Windows

### DISCUSSION THREAD

**Discussion by salric** [stoplightio/spectral#1974] [General]
A while ago, I used to run the spectral-cli.exe lint command on windows successfully (e.g. spectral-cli.exe lint swagger2model.json). I hadn't used it for a while, and so just downloaded the latest.  

Now, when I run that command, the response is "Invalid ruleset provided".

So, I suppose the default ruleset was automatically installed in the past.  How can I install the default ruleset JSON for swagger2/OAS2 on Windows?  What is the correct syntax in a Windows CMD window to specify that ruleset?

**Comment by philsturgeon** [comment]
It sounds like you were on v5 and now you're on v6. I'd start with a look at the migration guide. 

https://meta.stoplight.io/docs/spectral/ZG9jOjg2MDIwMDM-spectral-v5-to-v6-migration-guide

**Reply by salric** [reply]
The experience used to be just install the Windows version and run it from the command line. There was nothing else needed. It would be great if someone could provide an example of how to get to that point again. If it involves installing a ruleset or some other customization that was not needed before, give us a step-by-step that shows us how. The migration guide does not help me.  Thanks for responding.

**Reply by P0lip** [reply]
> The experience used to be just install the Windows version and run it from the command line. 

That's still the case. The only difference is that you need to have a ruleset specified, but you seem to have it in your project. Otherwise, you'd get a different message than "invalid ruleset provided". It'd be something along these lines `'No ruleset has been found. Please provide a ruleset using the --ruleset CLI argument, or make sure your ruleset file matches .?spectral.(js|ya?ml|json)`

Could you attach the ruleset here if you have such?

**Reply by salric** [reply]
It would just be the default Swagger 2 ruleset.
